backends/dpdk/dpdkHelpers.h

```diff
@@ -133,18 +133,34 @@ class BranchingInstructionGeneration {
     bool generate(const IR::Expression *, cstring, cstring, bool);
 };
 
+class TypeWidthValidator : public Inspector {
+    void postorder(const IR::Type_Varbits *type) override {
+        LOG3("Validating Type_Varbits: " << type);
+        if (type->size % 8 != 0) {
+            ::error(ErrorType::ERR_UNSUPPORTED,
+                    "%1% Maximum varbit width (%2%) not aligned to 8 bits",
```

Comment:
Why 'maximum' in the error message?

Reply:
I wanted to make it clear that this is checking the statically-declared maximum width of the varbit field which is stated in the definition of that field, f.e. if you have a header field defined as `varbit<304> data;`, this check only checks that the maximum width  `304` is aligned to 8 bits, not a dynamic width of the value which is parsed into that field.

The spec in section [7.1](https://p4.org/p4-spec/docs/P4-16-v1.2.2.html#sec-base-types) states:
> Bit-strings of dynamically-computed width with a fixed maximum width varbit<>

So I thought that it will be better to use also the word 'maximum'.

But I'm open to any other suggestion.

Reply:
Just put yourself in the shoes of a user who gets this error. I am not sure they will know what the maximum is taken over.
I would just remove that word.

Reply:
Ok, makes sense. Done.